src/euphorie/client/browser/templates/webhelpers.pt

```diff
@@ -165,22 +165,24 @@
               "
     >
 
-      <!-- Existing Measures -->
-      <fieldset class="measures-in-place pat-checklist pat-clone ${python:'pat-inject pat-autosubmit pat-subform' if (webhelpers.integrated_action_plan or webhelpers.use_training_module) else None}"
-                id="risk_identification_form"
-                disabled="${python:'disabled' if not webhelpers.can_edit_session else None}"
-                data-pat-clone="template: #in-place-measure-template; trigger-element: #add-in-place-measure-button; remove-behaviour: none"
-                data-pat-inject="${python:'source: #panel-action-plan::element; target: #panel-action-plan::element;' if webhelpers.integrated_action_plan else None}${python:' &amp;&amp; source: #page-module-training::element; target: #page-module-training::element;' if webhelpers.use_training_module else None}"
-      >
-        <tal:existing_measures condition="view/show_existing_measures">
+      <tal:measures define="
+                      measures view/get_existing_measures_with_activation;
+                    ">
+        <!-- Existing Measures -->
+        <fieldset class="measures-in-place pat-checklist pat-clone ${python:'pat-inject pat-autosubmit pat-subform' if (webhelpers.integrated_action_plan or webhelpers.use_training_module) else None}"
+                  id="risk_identification_form"
+                  disabled="${python:'disabled' if not webhelpers.can_edit_session else None}"
+                  data-pat-clone="template: #in-place-measure-template; trigger-element: #add-in-place-measure-button; remove-behaviour: none"
+                  data-pat-inject="${python:'source: #panel-action-plan::element; target: #panel-action-plan::element;' if webhelpers.integrated_action_plan else None}${python:' &amp;&amp; source: #page-module-training::element; target: #page-module-training::element;' if webhelpers.use_training_module else None}"
+                  tal:define="
+                    show_existing_measures view/show_existing_measures;
+                  "
+                  tal:condition="python:len(measures) > 0 and show_existing_measures"
```

Comment:
Shouldn't this be
```suggestion
                  tal:condition="show_existing_measures"
```
and `show_existing_measures` should take into account if you already have measures?

Reply:
I can no longer reproduce the problem I fixed here. It might have been a side effect. I have removed that change now again.